.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+		"dbaeumer.vscode-eslint", "vtfn.stylint", "eamodio.gitlens",
```

Comment:
"vtfn.stylint" looks like an outdated version, or not official?  Its last update was 2016:
https://marketplace.visualstudio.com/items?itemName=vtfn.stylint

My settings use stylelint.vscode-stylelint which looks more official and was last updated ~2 weeks ago:
https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint